docs/conf.py

```diff
@@ -84,7 +84,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = "en"
```

Comment:
not sure what this does?

Reply:
The new version of Sphix does not allow `None`, but previously `None` evaluated to `en` anyway.

https://github.com/sphinx-doc/sphinx/issues/10474